docs/tls/KUBERNETES-SECRETS.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes Secrets
+
+## Overview
+
+Kubernetes Secrets are used to provide the required certificates to NLK. There are two ways to create the Secrets:
+- Using `kubectl`
+- Using yaml files
+
+The filenames for the certificates created previously are required for both methods. The examples below assume the certificates were generated in `/tmp` 
```

Comment:
In this case "previously" isn't very helpful. Maybe link to the cert generation docs?